include/lwipopts.h

```diff
@@ -99,7 +102,7 @@ a lot of data that needs to be copied, this should be set high. */
 #define LWIP_ICMP 1
 
 /* ---------- DHCP options ---------- */
-#define LWIP_DHCP 1
+#define LWIP_DHCP 0
```

Comment:
Should we not use DHCP as default for now since we are still working out networking issues?